src/pages/shops.tsx

```diff
@@ -0,0 +1,56 @@
+import React from "react";
+import { useQuery } from "@tanstack/react-query";
+import axios from "axios";
+import Link from "next/link";
+import Head from "next/head";
+
+interface Shop {
+  name: string;
+  email: string;
+  phone: string;
+  description?: string;
+}
+
+export default function shops() {
+  const getShops = async () => {
+    const response = await axios.get("/api/shop");
+    return response.data.data;
+  };
+
+  const { data: shops } = useQuery(["shops"], getShops, {
+    staleTime: 30000,
+  });
+
+  return (
+    <div className="justify-top m-4 flex h-screen flex-col items-center font-fredoka">
+      <Head>
+        <link rel="shortcut icon" href="/images/Group 10.png" />
+        <title>Shops</title>
+      </Head>
+      <ul className="">
+        {shops?.map((shop: Shop, index: number) => {
+          return (
+            <li key={index} className="m-4 max-w-sm border p-4">
```

Comment:
I used index instead because time constraints but I know to always used Id, I think I had gap in thinking because when I deal with data queries in mongo, _id and id are two different things so I use index when testing things then figure out the rest afterwards.